tools/test-proxy/ci.yml

```diff
@@ -25,3 +25,4 @@ stages:
   - template: ../../eng/pipelines/templates/stages/archetype-sdk-tool-dotnet.yml
     parameters:
       ToolDirectory: test-proxy
+      DotNetCoreVersion: "5.0.100"
```

Comment:
I assume you did this to keep it focused on test-proxy only for now. Can you please file an issue and assign to @chidozieononiwu to update all the tools to be .NET 5.0